test/e2e/specs/a8c-for-agencies/signup__smoke.ts

```diff
@@ -32,77 +32,28 @@ describe( 'A4A > Signup: Smoke Test', function () {
 
 		// Enter first name
 		const firstName = 'John';
-		await page.getByLabel( 'First name' ).fill( firstName );
+		await page.getByPlaceholder( 'Your first name' ).fill( firstName );
```

Comment:
👋 The fact that `getByLabel` doesn't work anymore hints at an accessibility issue. All inputs must have an accessible label. @Automattic/a4a-genesis maybe? (Also relevant that this sign-up page likely falls under the EU Accessibility Act p58i-lv1-p2)

Reply:
Yes, agree the A4A contact page needs an overhaul and accessibility review. We can treat that as a new task. I'll follow up. Thanks, @mirka

Reply:
@mirka I reported to the A4A team in Linear and CC'd you to follow along. Thanks for your nudge!

Reply:
Thanks for the fix and nudge. We will get this fixed!